Mathlib/Algebra/CentralSimple/Basic.lean

```diff
@@ -0,0 +1,60 @@
+/-
+Copyright (c) 2024 Kevin Buzzard. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yunzhou Xie, Jujian Zhang, Kevin Buzzard
+-/
+
+import Mathlib.Algebra.CentralSimple.Defs
+import Mathlib.RingTheory.SimpleRing.Basic
+
+/-!
+# Central Simple Algebras
+
+In this file, we prove some basic results about central simple algebras over a field.
+
+## Main results
+
+- `Algebra.IsCentralSimple.center_eq_bot`: the center of a central simple algebra over a field `K`
+  is equal to `K`.
+- `Algebra.IsCentralSimple.self`: a field is a central simple algebra over itself.
+- `Algebra.IsCentralSimple.baseField_essentially_unique`: Let `D/K/k` is a tower of scalars where
+  `K` and `k` are fields. If `D` is central simple over `k`, `K` is isomorphic to `k`.
+-/
+
+universe u v
+
+namespace Algebra.IsCentralSimple
```

Comment:
Results in this file still holds for IsCentral algebras, I think.

Reply:
the last one needs non-tiriviality, other than that everything generalizes